api_tests/Create_my_account.yaml

```diff
@@ -0,0 +1,54 @@
+# Test generated by RoostGPT for test credwallet-integration-artillery-updated using AI Type Open AI and AI Model gpt-4
+
+
+config:
+  target: "{{ $processEnvironment.API_HOST }}"
+  phases:
+    - duration: 60
+      arrivalRate: 20
+  plugins:
+    expect: {}
+  payload:
+    path: "Create_my_account.csv"
+    fields:
+      - "email"
+    skipHeader: true
+  environments:
+    dev:
+      target: "http://localhost:8080"
+    prod:
+      target: "http://prod.host.com"
+scenarios:
+  - name: "Create my account"
+    flow:
+      - post:
+          url: "/accounts"
+          json:
+            email: "{{ email }}"
+          headers:
+            commit-hash: "{{ $processEnvironment.COMMIT_HASH }}"
+            Authorization: "{{ $processEnvironment.AUTH_TOKEN }}"
+          expect:
+            - statusCode: 201
+            - hasProperty: "accountId"
+            - hasHeader: "api-version"
+            - headerEquals: ["api-version", "1.0.0"]
```

Comment:
header assertions.

Reply:
* GET /me 

  ok statusCode 201 
  not ok hasProperty response body is not an object 
  expected: accountId
       got: response body is not an object
  ok hasHeader api-version 
  ok header api-version values equals 1.0.0 
  ok hasHeader commit-hash 
  not ok matchesRegexp false 
  expected: true
       got: false
  Request params:
    http://localhost:4012/accounts
    {
      "email": "paul@example.com"
    }
  Headers:
  access-control-allow-origin: *
  access-control-allow-headers: *
  access-control-allow-credentials: true
  access-control-expose-headers: *
  api-version: 1.0.0
  commit-hash: 928d28d
  content-type: text/plain
  date: Thu, 21 Dec 2023 07:57:04 GMT
  connection: keep-alive
  keep-alive: timeout=5
  content-length: 0
  Body:

  User variables:
    target: http://localhost:4012
    $environment: undefined
    email: paul@example.com
    $uuid: e6eba4af-fcf2-42d7-a715-8e0c9cf5a708
  ok statusCode 200 
  not ok hasProperty response body has no accountId property 
  expected: accountId
       got: response body has no accountId property
  ok hasProperty email 
  not ok equals email, susan@example.com 
  expected: all values to be equal
       got: email, susan@example.com
  ok hasProperty settings 
  ok hasProperty createdAt 
  ok hasProperty updatedAt 
  ok hasHeader api-version 
  ok header api-version values equals 1.0.0 
  Request params:
    http://localhost:4012/me
    ""
  Headers:
  access-control-allow-origin: *
  access-control-allow-headers: *
  access-control-allow-credentials: true
  access-control-expose-headers: *
  api-version: 1.0.0
  content-type: application/json
  content-length: 273
  date: Thu, 21 Dec 2023 07:57:04 GMT
  connection: keep-alive
  keep-alive: timeout=5
  Body:
{"id":"497f6eca-6276-4993-bfeb-53cbbbba6f08","did":"string","name":"string","image":"https://cdn.credentialregiater.com/2571c54e-2cde-4434-83f2-c329f74adc27.png","email":"user@example.com","settings":{},"createdAt":"2019-08-24T14:15:22Z","updatedAt":"2019-08-24T14:15:22Z"}
  User variables:
    target: http://localhost:4012
    $environment: undefined
    email: susan@example.com
    $uuid: c3086a98-6490-4191-bc0f-36350813999c
* POST /accounts 

  ok statusCode 201 
  not ok hasProperty response body is not an object 
  expected: accountId
       got: response body is not an object
  ok hasHeader api-version 
  ok header api-version values equals 1.0.0 
  ok hasHeader commit-hash 
  not ok matchesRegexp false 
  expected: true
       got: false
  Request params:
    http://localhost:4012/accounts
    {
      "email": "susan@example.com"
    }
  Headers:
  access-control-allow-origin: *
  access-control-allow-headers: *
  access-control-allow-credentials: true
  access-control-expose-headers: *
  api-version: 1.0.0
  commit-hash: 928d28d
  content-type: text/plain
  date: Thu, 21 Dec 2023 07:57:04 GMT
  connection: keep-alive
  keep-alive: timeout=5
  content-length: 0
  Body:

  User variables:
    target: http://localhost:4012
    $environment: undefined
    email: susan@example.com
    $uuid: 7b134f7b-d76c-4014-921d-8be0ba32959a